pkg/inject/inject.go

```diff
@@ -45,12 +46,21 @@ const (
 	envDestinationContext         = "LINKERD2_PROXY_DESTINATION_CONTEXT"
 	envDestinationProfileSuffixes = "LINKERD2_PROXY_DESTINATION_PROFILE_SUFFIXES"
 	envDestinationSvcAddr         = "LINKERD2_PROXY_DESTINATION_SVC_ADDR"
-
-	envIdentityDisabled = "LINKERD2_PROXY_IDENTITY_DISABLED"
-	identityDisabledMsg = "Identity is not yet available"
+	envDestinationSvcName         = "LINKERD2_PROXY_DESTINATION_SVC_NAME"
 
 	// destinationAPIPort is the port exposed by the linkerd-destination service
 	destinationAPIPort = 8086
+
+	envIdentityDisabled     = "LINKERD2_PROXY_IDENTITY_DISABLED"
+	envIdentityDir          = "LINKERD2_PROXY_IDENTITY_DIR"
+	envIdentityLocalName    = "LINKERD2_PROXY_IDENTITY_LOCAL_NAME"
+	envIdentitySvcAddr      = "LINKERD2_PROXY_IDENTITY_SVC_ADDR"
+	envIdentitySvcName      = "LINKERD2_PROXY_IDENTITY_SVC_NAME"
```

Comment:
Likewise, I don't see this in the [proxy source](https://github.com/linkerd/linkerd2-proxy/search?q=LINKERD2_PROXY_IDENTITY_SVC_NAME&unscoped_q=LINKERD2_PROXY_IDENTITY_SVC_NAME). I brought it up because all that other env vars are in src/app/config.rs, except these two.